subprojects/shipkit/src/test/groovy/org/shipkit/gradle/release/ReleaseNeededTaskTest.groovy

```diff
@@ -26,23 +26,30 @@ class ReleaseNeededTaskTest extends Specification {
 
-        null                | "master"  | false       | null       | ["", "diff"]      || true  // null commit msg
-        " "                 | "master"  | false       | null       | ["", "diff"]      || true  // only whitespaces in commit msg
-        "message"           | "master"  | false       | null       | ["", "diff"]      || true  // publications differ
+        null                 | "master"  | false       | null       | ["", "diff"]      || true  // null commit msg
+        " "                  | "master"  | false       | null       | ["", "diff"]      || true  // only whitespaces in commit msg
+        "message"            | "master"  | false       | null       | ["", "diff"]      || true  // publications differ
 
-        "[ci skip-release]" | "master"  | false       | null       | ["", "diff"]      || false // skip-release in commit msg
-        "message"           | "feature" | false       | null       | ["", "diff"]      || false // feature branch
-        "message"           | null      | false       | null       | ["", "diff"]      || false // null branch
-        "message"           | "master"  | true        | null       | ["", "diff"]      || false // pull request
-        "message"           | "master"  | false       | "true"     | ["", "diff"]      || false // SKIP_RELEASE set
-        "message"           | "master"  | false       | null       | ["", ""]          || false  // publications are the same
-        "message"           | "master"  | false       | null       | []                || true   // no comparison results
+        "[ci skip-release]"  | "master"  | false       | null       | ["", "diff"]      || false // skip-release in commit msg
+        "message"            | "feature" | false       | null       | ["", "diff"]      || false // feature branch
+        "message"            | null      | false       | null       | ["", "diff"]      || false // null branch
+        "message"            | "master"  | true        | null       | ["", "diff"]      || false // pull request
+        "message"            | "master"  | false       | "true"     | ["", "diff"]      || false // SKIP_RELEASE set
+        "message"            | "master"  | false       | null       | ["", ""]          || false  // publications are the same
+
+        "[ci force-release]" | "master"  | false       | null       | ["", ""]          || true  // force-release
+        "[ci force-release]" | "master"  | true        | null       | ["", ""]          || false // force-release
+        "[ci force-release]" | "master"  | false       | null       | ["", "diff"]      || true  // force-release
+        "[ci force-release]" | "master"  | true        | null       | ["", "diff"]      || false // force-release
+        "[ci force-release]" | "dev1"    | true        | null       | ["", "diff"]      || false // force-release
```

Comment:
I would add a case that `skipEnvVar` is true - e.g. building master with different Java versions. In that case the `[ci force-release]` command should be ignored.